src/components/Dialog/index.web.tsx

```diff
@@ -33,20 +33,17 @@ export function Outer({
   const t = useTheme()
   const {gtMobile} = useBreakpoints()
   const [isOpen, setIsOpen] = React.useState(false)
-  const [isVisible, setIsVisible] = React.useState(true)
   const {setDialogIsOpen} = useDialogStateControlContext()
 
   const open = React.useCallback(() => {
-    setIsOpen(true)
     setDialogIsOpen(control.id, true)
+    setIsOpen(true)
   }, [setIsOpen, setDialogIsOpen, control.id])
 
   const close = React.useCallback<DialogControlProps['close']>(
     cb => {
       setDialogIsOpen(control.id, false)
-      setIsVisible(false)
       setIsOpen(false)
-      setIsVisible(true)
```

Comment:
`isVisible` has a default value of `true` and here we never actually would have changed this to `false`.

Reply:
I also can't see a visual difference after removing this in the web version, so unsure if we need it. We're conditionally rendering in the dialog once `isOpen` changes to true, so we still get the `Animated.View` `entering` animations.

Reply:
State calls during events should be batched so I'm not sure there'd be a difference either way.

Reply:
Oh ya know what, this is leftovers from an exit animation we removed, def safe to remove